### PR TITLE
remove references

### PR DESCRIPTION
**Pull request by anurag-arjun**

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -286,15 +285,15 @@ medium.
 
 ## Receive AVL Testnet Tokens
 
-Head over to the [Polygon faucet](https://faucet.polygon.technology).
+Head over to the [faucet](https://faucet.polygon.technology).
```
Comment (prabal-banerjee) [inline docs/quick-start.md:288]: Same question.

File: docs/quick-start.md [hunk 2]
```diff
@@ -387,7 +386,7 @@ main().catch(console.error)
 Sample Result:
 
 ```
-You are connected to chain Avail-Testnet using Polygon Avail Node v3.0.0-6c8781e-x86_64-linux-gnu
+You are connected to chain Avail-Testnet using Avail Node v3.0.0-6c8781e-x86_64-linux-gnu
```
Comment (prabal-banerjee) [inline docs/quick-start.md:389]: We still have Polygon on the node version on testnet and devnet. We need to update that.

**Review by prabal-banerjee — APPROVED** [review]

File: docs/quick-start.md [hunk 3]
```diff
@@ -91,7 +90,7 @@ copy your address.  Alternatively, you can copy the address manually.
 
 <img src={useBaseUrl("img/avail/account-icon.png")} align= "center" width="100%" height="100%"/>
 
-Head over to the [Polygon faucet](https://faucet.polygon.technology).
+Head over to the [faucet](https://faucet.polygon.technology).
```
[the remark this answers is not in the document]
Comment (vthunder) [reply docs/quick-start.md:93]: Will the Polygon one continue to work?

**Review by vthunder — APPROVED** [review]
lgtm

File: docs/quick-start.md (hunk 2 above)
Comment (vthunder) [reply docs/quick-start.md:389]: We should cut a new release for this, even if that’s the only change. @jakubcech let’s discuss early next week and get it on the board

**Pull request merged**